### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Metra25**
- [DEMO LINK](https://Metra25.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://Metra25.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,9 +6,51 @@
...
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body id="home">
+    <header class="header">
+      <a href="#home">
+        <img class="img"
```
Comment (Esceype) [inline src/index.html:17]: And change the name of your class
```suggestion
        <img
          class="img"
```

File: src/index.html [hunk 2]
```diff
@@ -6,9 +6,51 @@
...
+          </li>
+          <li class="nav-item">
+            <a href="#Smartphones" class="nav-link">Smartphones</a>
+          </li>
+          <li class="nav-item">
+            <a
+            data-qa="hover"
+            href="#Laptops & Computers"
+            class="nav-link">Laptops & Computers
+            </a>
```
Comment (Esceype) [inline src/index.html:38]: Checklist 13

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,72 @@
...
+  font-weight: 500;
+  text-decoration: none;
+  font-size: 12px;
+  color: black;
+  margin-left: 20px;
+}
+
+.nav-link:hover {
+  color: var(--link-color);
+}
```
Comment (Esceype) [inline src/style.css:56]: Checklist 10, 11

**Review by Esceype — CHANGES_REQUESTED** [review]
Overall GJ !!!! Add demo link and report tests link to the description of PR

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,72 @@
...
+  color: var(--link-color);
+  margin-left: 0;
+}
+
+.is-active::after {
+  position: absolute;
+  content: "";
+  display: block;
+  background-color: var(--link-color);
+  bottom: 0;
```
[the remark this answers is not in the document]
Comment (Metra25) [reply src/style.css:61]: Не совсем понял, чего тут много))

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,65 @@
...
+}
+
+.nav__link {
+  display: block;
+  padding: 23px 0 ;
+  font-weight: 500;
+  text-decoration: none;
+  font-size: 12px;
+  color: black;
+  margin-left: 20px;
```
Comment (xiimiik) [inline src/style.css:43]: edge elements should not be indented

File: src/style.css [hunk 6]
```diff
@@ -1,3 +1,65 @@
...
+.header__logo {
+  margin: 10px 0;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
```
Comment (xiimiik) [inline src/style.css:28]: ul have padding also

**Review by xiimiik — CHANGES_REQUESTED** [review]
I see code changes, but u forgot to deploy ur changes

File: src/style.css [hunk 7]
```diff
@@ -1,3 +1,67 @@
...
+}
+
+.nav__item {
+  position: relative;
+  text-transform: uppercase;
+}
+
+.nav__link {
+  display: block;
+  padding: 23px 0 ;
```
Comment (mykhalenych) [inline src/style.css:38]: Screenshot 2023-08-08 at 18 41 58

check your padding

**Review by mykhalenych — CHANGES_REQUESTED** [review]

**Pull request closed without merging**